internal/util/update/update_test.go

```diff
@@ -678,6 +678,46 @@ func TestCommand_Run_failInvalidPath(t *testing.T) {
 	}
 }
 
+// TestCommand_Run_failInvalidRepo verifies Run fails if the repo is invalid
+func TestCommand_Run_failInvalidRepo(t *testing.T) {
+	for _, strategy := range kptfilev1alpha2.UpdateStrategies {
```

Comment:
We don't really need to test for each strategy here. This functionality is independent of which strategy is being used.

Reply:
Removed.